designcompose/src/main/java/com/android/designcompose/Layout.kt

```diff
@@ -490,7 +484,22 @@ internal fun calcLayoutInfo(
             )
         }
     } else {
-        return LayoutInfoAbsolute(modifier)
+        var horizontalScroll = false
+        var verticalScroll = false
+        when (view.scroll_info.overflow) {
+            is OverflowDirection.VERTICAL_SCROLLING -> {
+                verticalScroll = true
+            }
+            is OverflowDirection.HORIZONTAL_SCROLLING -> {
+                horizontalScroll = true
+            }
+            is OverflowDirection.HORIZONTAL_AND_VERTICAL_SCROLLING -> {
```

Comment:
We don't have any way of surfacing this to the user, right? That we don't support both?